packages/react/src/migrations/update-8-3-0/update-8-3-0.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree
+} from '@angular-devkit/schematics';
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+import {
+  addDepsToPackageJson,
+  formatFiles,
+  insert,
+  updateJsonInTree
+} from '@nrwl/workspace';
+import {
+  createSourceFile,
+  isImportDeclaration,
+  isStringLiteral,
+  ScriptTarget
+} from 'typescript';
+import { ReplaceChange } from '@nrwl/workspace/src/utils/ast-utils';
+import { relative } from 'path';
+import { testingLibraryVersion } from '../../utils/versions';
+
+const ignore = require('ignore');
+
+export default function update(): Rule {
+  return chain([
+    displayInformation,
+    updateDependencies,
+    updateImports,
+    formatFiles()
+  ]);
+}
+
+function displayInformation(host: Tree, context: SchematicContext) {
+  context.logger.info(stripIndents`
+    React Testing Library has been repackaged as \`@testing-library/react\` as of version 8.
+    
+    We are replacing your \`react-testing-library\` imports with \`@testing-library/react\`.
+    
+    See: https://github.com/testing-library/react-testing-library/releases/tag/v8.0.0
+  `);
+}
+
+function updateDependencies() {
```

Comment:
Please update the other dependencies as well such as `react`, `@types/react` etc.

Note: Use a hard coded version here in case people are not updating to the latest version it still works properly.